assets/src/components/detours/diversionPage.tsx

```diff
@@ -47,6 +48,24 @@ export const DiversionPage = ({
   } = useDetour(originalRoute.routePatternId)
 
   const [textArea, setTextArea] = useState("")
+
+  useEffect(() => {
+    setTextArea(
+      [
+        "Detour:",
+        `${originalRoute.routeName} ${originalRoute.routeDescription} from`,
+        originalRoute.routeOrigin,
+        originalRoute.routeDirection,
+        ,
+        "Turn-by-Turn Directions:",
+        ...(directions?.map((v) => v.instruction) ?? []),
+        ,
+        `Missed Stops (${missedStops?.length}):`,
+        ...(missedStops?.map(({ name }) => name) ?? ["no stops"]),
+      ].join("\n")
```

Comment:
**suggestion (non-blocking):** I think there's enough logic here that it _might_ make sense to put in a private helper function, but I do realize it would have to take quite a few arguments so it might be kind of unwieldy.

Reply:
I'll keep this in mind for after I add connection points 👍🏻